Formula/openjpeg.rb

```diff
@@ -1,36 +1,31 @@
 class Openjpeg < Formula
   desc "Library for JPEG-2000 image manipulation"
-  homepage "http://www.openjpeg.org"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/o/openjpeg/openjpeg_1.5.2.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/o/openjpeg/openjpeg_1.5.2.orig.tar.gz"
-  sha256 "aef498a293b4e75fa1ca8e367c3f32ed08e028d3557b069bf8584d0c1346026d"
-  revision 1
-
-  head "https://github.com/uclouvain/openjpeg.git", :branch => "openjpeg-1.5"
+  homepage "http://www.openjpeg.org/"
+  url "https://github.com/uclouvain/openjpeg/archive/v2.1.1.tar.gz"
+  sha256 "82c27f47fc7219e2ed5537ac69545bf15ed8c6ba8e6e1e529f89f7356506dbaa"
 
   bottle do
     cellar :any
-    sha256 "50be982ed3846844ac93a34ebd159c680bed484f96b6c0ea274c2e88cc5501de" => :el_capitan
-    sha256 "c9cc5b4f37fdf8b8fc1b043597ab6ac4aa30bfa8fde94b7b0d67a30f03a3cb1b" => :yosemite
-    sha256 "92badbf4968bfda26318855b28940f564e60c701fe3b4e29bc4f173748da7476" => :mavericks
-    sha256 "a30aa5b0a7ebcc1daba910671183084d69afb1d30cb85bfeb8b213f8e7a617d7" => :mountain_lion
+    sha256 "f40e41ca1b1ee59149411ca5054bb7aded1323f64d5b84a30ff88377d946610d" => :yosemite
+    sha256 "9004d4427b26fb5d5178c5c8e942cc225d6e5e9a2439d63fb74c4022ce87544a" => :mavericks
+    sha256 "f1c0a8432b57e712d0281dc0093a1d072f6f51f3c9e81eb533faa182956d46e3" => :mountain_lion
   end
 
+  option "with-doc", "Build HTML documentation."
```

Comment:
ok. it's also worth mentioning this also builds man pages for the binaries. Both man1 and man3. Total size the share folder is 12.4 MB (not MiB).